.github/workflows/kicad.yml

```diff
@@ -47,10 +47,10 @@ jobs:
       #     out_dir: kicad-artifacts
       #     versioning: false
 
-      - name: comment
-        uses: marocchino/sticky-pull-request-comment@v2
-        with:
-          path: build/comment.md
+      # - name: comment
+      #   uses: marocchino/sticky-pull-request-comment@v2
+      #   with:
```

Comment:
is this still broken

Reply:
This is the comment from Github Actions that's currently broken. It can be added back if/when we fix that action and find a place to store files (e.g. Digital Ocean).